tzatziki-http/src/test/resources/com/decathlon/tzatziki/steps/http.feature

```diff
@@ -1311,7 +1311,7 @@ Feature: to interact with an http service and setup mocks
     Then the received body on server socket checksum is equal to <gzipEncodedBodyChecksum>
 
     Given that we listen for incoming request on a test-specific socket
-    When we send on "http://127.0.0.1:{{{[serverSocket.localPort]}}}":
+    When we send on "/":
```

Comment:
In my understanding, this PR implements a new behaviour/feature. It does not change an existing one. Consequently, I would expect to see an additional test, dedicated to this change, instead of a modified existing test.

Reply:
You're right, I was struggling an easy way to test it but I figured out I think, testing the whole rewriting url feature.